zmon-controller-app/src/main/java/org/zalando/zmon/api/domain/AlertResults.java

```diff
@@ -0,0 +1,11 @@
+package org.zalando.zmon.api.domain;
+
+import java.util.List;
+
+public class AlertResults {
```

Comment:
What's the reason for creating a wrapper class for the list of objects?

Reply:
The reason is to have an object as a top level structure, in order to allow future extensions. See https://opensource.zalando.com/restful-api-guidelines/#167.